### PR TITLE
Icds changes

### PR DESCRIPTION
**Pull request by lwyszomi**
@sheelio 
https://manage.dimagi.com/default.asp?263448#1406263
https://manage.dimagi.com/default.asp?263551#1405676
https://manage.dimagi.com/default.asp?263601#1406204
https://manage.dimagi.com/default.asp?263602#1406207
https://manage.dimagi.com/default.asp?263630#1406351

### CODE REVIEW

File: custom/icds_reports/reports/awc_reports.py [hunk 1]
```diff
@@ -1086,11 +1092,23 @@ def get_status(value):
...
+                'underweight',
+                'Normal weight for age'
+            ),
             recorded_weight=row_data.recorded_weight or 0,
             recorded_height=row_data.recorded_height or 0,
-            stunning=get_status(row_data.current_month_stunting),
-            wasting=get_status(row_data.current_month_wasting),
+            stunning=get_status(
+                row_data.current_month_stunting,
+                'stunned',
```
Comment (sheelio) [inline custom/icds_reports/reports/awc_reports.py]: Should this be stunned or stunted?

**Comment by fyniac** [comment]
@snopoke

**Review by snopoke — APPROVED** [review]
Pending @sheelio comment

File: custom/icds_reports/reports/awc_reports.py (hunk 1 above)
Comment (snopoke) [reply custom/icds_reports/reports/awc_reports.py]: @lwyszomi

Comment (lwyszomi) [reply custom/icds_reports/reports/awc_reports.py]: @snopoke @sheelio Yes, should be stunted, sorry for the typo.

Comment (lwyszomi) [reply custom/icds_reports/reports/awc_reports.py]: I sent new commit

**Pull request merged**